### DISCUSSION TITLE
purchased but can't be install

### DISCUSSION THREAD

**Discussion by CaptainGod** [wstyres/Zebra#1974] [Q&A]
Hi. I bought Crane through Zbra and Havoc. Thank you for making this attractive plugin. However, I may not be able to install it properly. In Havoc it was purchased, but in ZBRA it remained unpurchased and could not be installed. Can you help me solve this problem? Thank you very much!

**Comment by kirb** [comment]
I think this is an issue with Havoc rather than Zebra. Logging out of Havoc via Sources → Havoc → My Account → Sign Out, then logging back in may fix it. You could also try Home → settings cog → Reset → Clear Sources Cache. If that still doesn’t work, I’d recommend getting in touch with them at support@havoc.app.
